src/components/DocumentList.tsx

```
@@ -156,37 +155,37 @@ const DocumentList: React.FC<DocumentListProps> = ({ documents }) => {
-    ];
+      const images: string[] = [];
 
-    possibleKeys.forEach((key) => {
-      const content = decodedData?.credentialSubject?.[key]?.content;
-      const mimeType =
-        decodedData?.credentialSubject?.[key]?.mimetype ?? "image/png";
-
-      if (content && isBase64(content)) {
-        images.push(`data:${mimeType};base64,${content}`);
+      if (credentialSubject && typeof credentialSubject === "object") {
+        Object.values(credentialSubject).forEach((entry: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)